gensim/models/doc2vec.py

```diff
@@ -808,6 +809,50 @@ def delete_temporary_training_data(self, keep_doctags_vectors=True, keep_inferen
         if self.docvecs and hasattr(self.docvecs, 'doctag_syn0_lockf'):
             del self.docvecs.doctag_syn0_lockf
 
+    def save_word2vec_format(self, fname, doc_vec=True, word_vec=False, prefix='dt_', fvocab=None, binary=False):
```

Comment:
Thinking more about names: 

* to acknowledge the doc-vectors aren't necessarily one-per-doc, but one-per-doctag, and to follow the convention elsewhere, let's enable with `doctag_vec=True`, rather than `doc_vec=True`.
* let's make the default prefix even weirder and less-at-risk of collision with any real tokens. In Mikolov's example sentence-vectors scripts, he prefixed those vector-keys with `*_`. So let's use `*dt_` as the default prefix.